tests/references/test_api.py

```diff
@@ -280,22 +280,15 @@ async def test_edit(data_type, error, mocker, snapshot, fake2, spawn_client, res
         )
         return
 
-    if error == "422":
-        await resp_is.invalid_input(
```

Comment:
Missing check of status code. I suggest having two error tests like `400_duplicates`, `400_invalid_input`.